eo-maven-plugin/src/main/java/org/eolang/maven/OyRemote.java

```diff
@@ -40,35 +40,84 @@ final class OyRemote implements Objectionary {
     /**
      * The address template.
      */
-    private final String template;
+    private final UrlOy template;
 
     /**
      * Constructor.
      * @param hash Commit hash
      * @throws IOException if fails.
      */
     OyRemote(final CommitHash hash) throws IOException {
-        this.template = String.format(
-            "https://raw.githubusercontent.com/objectionary/home/%s/objects/%%s.eo",
+        this.template = new UrlOy(
+            "https://raw.githubusercontent.com/objectionary/home/%s/objects/%s.eo",
             hash.value()
         );
     }
 
     @Override
     public String toString() {
-        return this.template;
+        return this.template.toString();
     }
 
     @Override
     public Input get(final String name) throws MalformedURLException {
-        final URL url = new URL(
-            String.format(this.template, name.replace(".", "/"))
-        );
+        final URL url = this.template.get(name);
         Logger.debug(
             this, "The object '%s' will be pulled from %s...",
             name, url
         );
         return new InputOf(url);
     }
 
+    /**
+     * Objectionary URL template.
+     *
+     * @since 1.0
+     */
+    public static class UrlOy {
+
+        /**
+         * URL template. Expects two placeholders in terms of
+         * {@link String#format(String, Object...)}: 1st for hash,
+         * 2nd for program name.
```

Comment:
@mximp It's better to create a default constructor with hardcoded value:
```java
UrlOy( final String hash) {
      this("https://raw.githubusercontent.com/objectionary/home/%s/objects/%s.eo",hash);
}
```
And you won't need that explanation in the `JavaDoc`, moreover you will be able to restrict the main constructor with `private` modifier

Reply:
@volodya-lombrozo I would not agree here. I believe that URL template value belongs (i.e. should be defined) by `OyRemote` object, not by `UrlOy`. It is specific to particular Objectionary. 
To illustrate what I mean: would you define any default value for plain `java.util.URL` ?

Reply:
`java.util.URL` != `UrlOy`. We deal with a concrete URL. In other words, would you define any default value for plain `com.google.GoogleURL` class, for example? I would do :) Ok, I don't insist. Leave it as is, it was just a suggestion.